### DISCUSSION TITLE
Build for ARM64 machines?

### DISCUSSION THREAD

**Discussion by skrble** [Tichau/FileConverter#468] [Ideas]
Hi, as the new ARM64 powered Windows machines are finally entering the market, could you please compile an ARM build for us? That would be very nice as the current emulated x64 build simply runs quite slow on ARM machines. Thank you for an eventual reply. Cheers!

**Comment by whoehenrieder** [comment]
I'd have to also request an ARM64 build (if possible).  I was trying to convert some Video files using my new SnapDragon Lenovo laptop - and the conversion rate (going from MP4 to MKV or AVI) was SOOO SLOWWWW - I thought the program wasn't even working.  As ARM Windows machines become more and more popular - this will be a bigger request.
